CvPoint object_center
CvLine[] line_list
CvCircle[] circle_list